recipes/dd-agent-install-linux.rb

```diff
@@ -0,0 +1,43 @@
+#
+# Cookbook Name:: datadog
+# Recipe:: dd-agent-install-linux
```

Comment:
The recipe name here gets lengthy.

Please use something like `_install-linux.rb` so the final name will look like:

    include_recipe 'datadog::_install-linux' unless platform_family? 'windows'